src/libraries/Microsoft.Extensions.Configuration.Binder/Microsoft.Extensions.Configuration.Binder.sln

```diff
@@ -110,6 +110,10 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ref", "ref", "{F43534CA-C419-405E-B239-CDE2BDC703BE}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{94EEF122-C307-4BF0-88FE-263B89B59F9F}"
+	ProjectSection(SolutionItems) = preProject
+		..\Directory.Build.props = ..\Directory.Build.props
+		..\Directory.Build.targets = ..\Directory.Build.targets
+	EndProjectSection
```

Comment:
Easy to revert, but added these to lend visibility to the props/targets that affect compilation (i.e. searching for `SkipIncludeNullableAttributes` finds all instances).

Reply:
Please revert all solution changes. Thanks!